git-hooks/pre-push

```diff
@@ -4,26 +4,29 @@
 
 set -e
 
-protected_branch="main"
+protected_branch="refs/heads/main"
+current_branch="$(git symbolic-ref HEAD)"
+
+# Only run this hook for the "main" branch
+if [[ "$current_branch" != "$protected_branch" ]]; then
+    exit 0
+fi
```

Comment:
I found when pushing this branch that the hook caused git to fail on pushing a new, _unprotected_ branch. These changes relate to that.